### PR TITLE
feat: network list and status check tauri commands

### PR DESCRIPTION
**Pull request by alex1307**
Description

    Implement api.rs module which provides the contract between Tauri UI and underlying backend services.
    Implement network_list tauri command and apply the same approach to image_list command.
    Move the commands to api module which should handle the contract between UI and backend.
    Implement health_check endpoint providing the actual container's statuses read from Docker.
    Separate the Container states from the workspace configuration.

    Motivation and Context
    The Container State struct should be used as a binding between image (ImageType) and docker containers(name, id).
    The actual docker status should be read from docker with the provided attributes(container id for example).
    Health check

    How Has This Been Tested?
    Container module provides create, update and delete test cases.
    То test health check I run docker and uncommented the test_health_check in module health_check.rs
    I run the app and check that base node and tor images are being started correctly.

### CODE REVIEW

File: applications/launchpad/backend/tauri.conf.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "version": "1.1.0"
   },
   "build": {
-    "distDir": "../gui-react/build",
```
Comment (tarnas14) [inline applications/launchpad/backend/tauri.conf.json]: please remove this change before merge

**Comment by tarnas14** [comment]
I read through the code, I don't know enough rust yet to make a meaningful contribution to CR, I have a question, though:

do I understand correctly that we will be able to `invoke('health_check', 'Tor')` to get a status of Tor container?
and values that we pass should be stringified versions of ImageType enum?

File: applications/launchpad/backend/tauri.conf.json (hunk 1 above)
Comment (alex1307) [reply applications/launchpad/backend/tauri.conf.json]: I reverted the change back

**Comment by tomaszantas** [comment]
#200 will resolve the issue of missing `gui-react/build`

**Comment by tomaszantas** [comment]
I'm going to switch back to building only the launchpad jobs. Here is the prove that jobs are passed as expected:

image

image

**Review by tarnas14 — DISMISSED** [review]

**Review by tarnas14 — APPROVED** [review]

**Pull request merged**